README.md

```diff
@@ -4,12 +4,31 @@ Notation is a project to add signatures as standard items in the registry ecosys
 
 ## Table of Contents
 
+- [Notation Quick Start](#notation-quick-start)
 - [Branch](#branch)
 - [Core Documents](#core-documents)
 - [Community](#community)
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
+## Notation Quick Start
+
+- Install the Notation CLI from [Notation Releases][notation-releases]  
+  ```bash
+  curl -Lo notation.tar.gz https://github.com/shizhMSFT/notation/releases/download/v0.5.2/notation_0.5.2_linux_amd64.tar.gz
+  tar xvzf notation.tar.gz -C ~/bin notation
+  ```
+- Build, Push, Sign, Verify the `net-monitor` software
+  ```bash
+  export IMAGE=localhost:5000/net-monitor:v1
+  docker build -t $IMAGE https://github.com/wabbit-networks/net-monitor.git#main
+  docker push $IMAGE
+  notation cert generate-test --default --trust "wabbit-networks-dev"
```

Comment:
It's not clear what --default and --trust do, and what is wabbit-networks-dev. We can add comments where we want to explain what a particular notation command does.

Reply:
This was the quick-start, which is an abbreviated version of: #78 
This is the challenge with having required parameters, for good security reasons, while enabling quick-starts.
What would you suggest to keep that balance?

Reply:
The other docker commands are straightforward and probably well known to the reader. Where we introduce the notation commands, it would be nice to have a one line comments explaining what the specific command does.
```suggestion
  # Generate a test certificate, make it the default for signing, and add it as a trusted identity for verification
  notation cert generate-test --default --trust "wabbit-networks-dev"
```